README.md

```diff
@@ -1,79 +1,114 @@
-Takeaway Challenge
-==================
+# Takeaway Challenge #
+
+The client request to have a software to be able to make a user order some meal which will be delivered to his place later on.<br/>
+The user have to be able to see the menu with the dish name and price, and to select the wanted amount of the desired dishes.<br/>
+The user may be able to see the total of his order.<br/>
+The user may receive a text to let him know that his order have been placed and will be delivered in less that an hour from the moment finalised the order.<br/>
+
+Based on the user story, I have decided to create 5 classes and 1 module :<br/>
+
+1. The class menu, which is the menu itself containing an array of hashes with the different dish available and their price.<br/>
+
+2. The class order, where the user will be able to select the dishes wanted and their quantity.<br/>
+This will be done until the user do not want to select any more item.<br/>
+Afterward the basket will be created.<br/>
+
+3. The class summary which will make the total of the basket of the user, and show him a summary of his order.<br/>
+
+4. The class text which is responsible for sending a text to the user when he is finalising his order.<br/>
```

Comment:
Good breakdown of your program, including class name and responsibilities makes it clear how everything works/fits together. Nice one 👍